src/modules/fancyzones/editor/FancyZonesEditor/App.xaml.cs

```diff
@@ -21,6 +21,32 @@ namespace FancyZonesEditor
     /// </summary>
     public partial class App : Application
     {
+        // Localizable strings
+        private const string CrashReportMessageBoxTextPart1 = "Error logged to ";
+        private const string CrashReportMessageBoxTextPart2 = "Please report the bug to ";
+        private const string CrashReportMessageBoxCaption = "Error";
+
+        // Non-localizable strings
+        private const string CrashReportMessageBoxCaptionFZEditor = "FancyZones Editor ";
+        private const string CrashReportLogFile = "FZEditorCrashLog.txt";
+        private const string PowerToysIssuesURL = "https://aka.ms/powerToysReportBug";
+
+        private const string CrashReportExceptionTag = "Exception";
+        private const string CrashReportSourceTag = "Source: ";
+        private const string CrashReportTargetAssemblyTag = "TargetAssembly: ";
+        private const string CrashReportTargetModuleTag = "TargetModule: ";
+        private const string CrashReportTargetSiteTag = "TargetSite: ";
+        private const string CrashReportEnvironmentTag = "Environment";
+        private const string CrashReportCommandLineTag = "* Command Line: ";
+        private const string CrashReportTimestampTag = "* Timestamp: ";
+        private const string CrashReportOSVersionTag = "* OS Version: ";
+        private const string CrashReportIntPtrLengthTag = "* IntPtr Length: ";
+        private const string CrashReportx64Tag = "* x64: ";
+        private const string CrashReportCLRVersionTag = "* CLR Version: ";
+        private const string CrashReportAssembliesTag = "Assemblies - ";
+        private const string CrashReportDynamicAssemblyTag = "dynamic assembly doesn't has location";
```

Comment:
Originally I didn't notice the grammatical error in this one, it should be `have` not `has`

Reply:
Done.